webpack/scenes/RedHatRepositories/components/__tests__/RepositorySetRepositories.test.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import thunk from 'redux-thunk';
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import configureMockStore from 'redux-mock-store';
+import RepositorySetRepositories from '../RepositorySetRepositories';
+
+const mockStore = configureMockStore([thunk]);
+const store = mockStore({ katello: { redHatRepositories: { repositorySetRepositories: [] } } });
+
+describe('RepositorySetRepositories Component', () => {
+  it('sorts repos correctly', async () => {
+    const shallowWrapper = shallow(<RepositorySetRepositories
+      store={store}
+      contentId={1}
+      productId={2}
+      type="foo"
+    />);
+
+    const repos = [
+      { arch: 'x86_64', releasever: '5.11' },
+      { arch: 'x86_64', releasever: '7Server' },
+      { arch: 'x86_64', releasever: '7.10' },
+      { arch: 'x86_64', releasever: '7.1' },
+      { arch: 'i386', releasever: '5.11' },
+      { arch: 'i386', releasever: '5Workstation' },
+      { arch: 'x86_64', releasever: '7.11' }];
+
+    const result = shallowWrapper.dive().instance().sortedRepos(repos);
+
+    const expectedIndices = [1, 5, 6, 2, 3, 0, 4];
+
+    expectedIndices.forEach((expected, i) => {
+      expect(result[i]).toEqual(repos[expected]);
+    });
+    
+    expect(toJson(shallowWrapper)).toMatchSnapshot();
```

Comment:
it really should be in its own `its` block, which means you would have to split out `shallowRender` too. If you want to keep it in this `its` block, just adjust the description to `sorts repos correctly and renders` or something like that.